shell/common/mac/main_application_bundle.mm

```diff
@@ -30,7 +31,9 @@ bool HasMainProcessKey() {
 
   // Up to Contents.
   if (!HasMainProcessKey() &&
-      base::EndsWith(path.value(), " Helper", base::CompareCase::SENSITIVE)) {
+      (base::EndsWith(path.value(), " Helper", base::CompareCase::SENSITIVE) ||
+       base::EndsWith(path.value(), content::kMacHelperSuffix_renderer,
```

Comment:
`kMacHelperSuffix_plugin` not allowed?

Reply:
@nornagon that I know of we are not using the plugin helper; though I guess we could add it here for completeness

Reply:
@jkleinsc it will be used when pepper plugins are loaded.

Reply:
@deepak1556 ah I forgot about those.  I'll add that conditional.